### ISSUE TITLE
Accessibility issue: Page should contain a level-one heading on /az_core/mobile_nav_callback/menu_link_content%3A4da80941-7bc8-4545-910a-ef61fdc62e81

### ISSUE THREAD

**Issue by accesswatch** [accesswatch/qs-proposal#57]
## What
An accessibility scan flagged the element `<html>` because page should contain a level-one heading. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/page-has-heading-one?application=playwright.

[View screenshot](https://github.com/accesswatch/qs-proposal/blob/gh-cache/.screenshots/c4c7e3e6-1cdf-436d-9486-2922b6c5e6d7.png)

To fix this, ensure that the page, or at least one of its frames contains a level-one heading.

Specifically:

Fix all of the following:
-   Page must have a level-one heading

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
